content/neuvector-prime/azure/contents.lr

```diff
@@ -154,13 +167,27 @@ Important! Do not change or save the cluster configuration settings for Fullname
 
    !["Keyvaluepair config"](./keyvaluepairconfig.png)
 
+## Auto upgrade minor version
+
+By default the Kubernetes Service Extension "Auto upgrade minor version" is Enabled as part of NeuVector offer
+deployment. This will allow NeuVector to regularly update its CVE database.
```

Comment:
By default, the Kubernetes Service Extension property "Auto upgrade minor version" is Enabled during the NeuVector offer deployment. This will allow NeuVector to regularly update its CVE database.

Reply:
By default the Kubernetes Service Extension property "Auto upgrade minor version" is enabled during the NeuVector offer deployment. This will allow NeuVector to regularly update its CVE database.

Click "Refresh" in the Extension + application screen to view the latest Version.